src/index.ts

```diff
@@ -19,6 +19,11 @@ export interface FlashbotsBundleTransaction {
   signer: Signer
 }
 
+export interface FlashbotsOptions {
+  minTimestamp: number,
```

Comment:
We probably want to support people sending min or max by themselves? We should make the object keys optional. The logic below allows them to be undefined, but the type system does not.

Reply:
yep - missing the `?`